assets/src/edit-story/components/colorPicker/currentColorPicker.js

```diff
@@ -70,8 +70,16 @@ const HueWrapper = styled.div`
 `;
 
 const AlphaWrapper = styled.div`
-  background: #fff;
   ${wrapperCSS}
+  div:first-child div:first-child div:first-child {
```

Comment:
@barklund It looks like it's not really possible to overwrite the default opacity line background in a clean way -- this does it but it's not nice really. An alternative would be doing a custom opacity (doesn't feel like worth it just for the transparent background, another alternative would be revisiting the transparent bg with UX. Thoughts?

Reply:
There's always gonna be conflicts with third-party components, so this works for me.